config/initializers/aws.rb

```diff
@@ -1,15 +1,16 @@
 CarrierWave.configure do |config|
-  config.fog_provider = 'fog/aws'                     # required
-  config.fog_credentials = {
-    provider:              'AWS',                        # required
-    aws_access_key_id:     ENV['AWS_ACCESS_KEY'],            # required
-    aws_secret_access_key: ENV['AWS_SECRET_KEY'],                        # required
-    region:                'us-west-2',
-    path_style:            true            
-    # host:                  's3.example.com',             # optional, defaults to nil
-    # endpoint:              'https://s3.example.com:8080' # optional, defaults to nil
-  }
-  config.fog_directory  = 'servicealliance'                          # required
-  config.fog_public     = false                                        # optional, defaults to true
-  config.fog_attributes = { 'Cache-Control' => "max-age=#{365.day.to_i}" } # optional, defaults to {}
+  if Rails.env.production?
```

Comment:
@skatkov Sorry to bang on about this one...

How about setting a config variable in the environments, so for example in `config/environments/development.rb` we have:
```
config.carrierwave_storage = :file
```
and in `production.rb` we set this to `:fog`, then we can change the `storage` command in the uploader to `Rails.application.config.carrierwave_storage`, and the conditional in the `carrierwave.rb` initializer to use this value instead of the running environment.

Sorry if this seems like a small point, but I do feel it's better to store the environment configuration in the relevant environment files.